contrib/obs-pipewire-audio-capture/template.py

```diff
@@ -0,0 +1,23 @@
+pkgname = "obs-pipewire-audio-capture"
+pkgver = "1.1.5"
+pkgrel = 0
+build_style = "cmake"
+hostmakedepends = [
+    "cmake",
+    "ninja",
+    "pkgconf",
+]
+makedepends = [
+    "obs-studio-devel",
+    "pipewire-devel",
+]
+depends = [
+    "obs-studio",
+    "wireplumber",
+]
```

Comment:
```suggestion
```

obs is already linked and don't need the other one

Reply:
Does having xxx-devel in makedepends automatically put xxx in depends?

Reply:
nope